legend-depot-core-services/src/main/java/org/finos/legend/depot/services/api/versionedEntities/ManageVersionedEntitiesService.java

```diff
@@ -0,0 +1,22 @@
+//  Copyright 2021 Goldman Sachs
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+package org.finos.legend.depot.services.api.versionedEntities;
+
+import org.finos.legend.depot.services.api.entities.ManageEntitiesService;
+
+public interface ManageVersionedEntitiesService extends VersionedEntitiesService, ManageEntitiesService
```

Comment:
why does it extend the ManageEntitService?

Reply:
because we are providing the same methods as manageEntitiesService, but I think it doesn't make sense